ui/page/dcrdex/dcrdex_page.go

```diff
@@ -123,7 +125,7 @@ func (pg *DEXPage) Layout(gtx C) D {
 		msg = values.String(values.StrDexMainnetNotReady)
 	} else if !hasMultipleWallets {
 		msg = values.String(values.StrMultipleAssetRequiredMsg)
-	} else if !pg.AssetsManager.DexcInitialized() {
+	} else if !pg.AssetsManager.DEXCInitialized() || pg.CurrentPage() == nil {
```

Comment:
Should this page not close if `pg.CurrentPage()` is nil. If so, `Layout()` should just `return D{}` early on and not bother about anything else.

Reply:
> Should this page not close if pg.CurrentPage() is nil.

No. We still want to display at least a message, I think.